src/datasources/balances-api/coingecko-api.service.ts

```diff
@@ -369,4 +414,41 @@ export class CoingeckoApi implements IPricesApi {
       this.notFoundPriceTtlSeconds + CoingeckoApi.NOT_FOUND_TTL_RANGE_SECONDS,
     );
   }
+
+  /**
+   * Gets the in-memory cache key for the given cacheDir.
+   */
+  private getMemoryKey(cacheDir: CacheDir): string {
+    return `${cacheDir.key}:${cacheDir.field}`;
+  }
```

Comment:
Nit: shall we move this to the `CacheRouter`?

Reply:
Moved in [0cf8239](https://github.com/safe-global/safe-client-gateway/pull/2512/commits/0cf8239b6f15217eb2401d7ed840318c4b741771)